client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/patterns-data.ts

```diff
@@ -253,6 +254,41 @@ const useSectionPatterns = () => {
 				name: 'Two Column CTA',
 				category: callToAction,
 			},
+			{
+				id: 5645,
+				name: 'Four Recent Blog Posts',
+				category: posts,
```

Comment:
Question: Do we want to show these patterns in the build flow?

Reply:
@autumnfjeld, we could consider filtering patterns, or even categories, based on the user's goals/intent in the next iterations.

Reply:
> I think they don't hurt and wouldn't be a blocker anyways. Nobody has mentioned that they shouldn't be shown in other flows. I believe we want them in all flows because they are a core piece of WP.

I agree.